nekoyume/Assets/_Scripts/UI/Widget/LoginDetail.cs

```diff
@@ -143,7 +143,7 @@ public void BackToLogin()
             login.Show();
         }
 
-        private void Init(int index)
+        private async void Init(int index)
```

Comment:
```suggestion
        private async Task Init(int index)
```

Reply:
it can't change Task because `Init` using unityEvent callback. https://github.com/planetarium/NineChronicles/blob/2b6e962e6f3feeaf1a7cf5e83cbef78f0474a5bf/nekoyume/Assets/_Scripts/UI/Widget/LoginDetail.cs#L63